system/disk_usage.tf

```diff
@@ -0,0 +1,33 @@
+resource "datadog_monitor" "fs_free_inodes" {
+  name    = "Lack of count free inodes ${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "Lack of count free inodes ${var.free_inodes_time} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.instance_id.public_ip}"
+  query   = "avg(last_${var.free_inodes_time}):avg:system.fs.inodes.free{host:${data.aws_instance.monitored.instance_id}} by {device,host} < ${var.free_inodes_critical_state_value}"
+
+  thresholds {
+    ok       = "${var.free_inodes_ok_state_value}"
+    warning  = "${var.free_inodes_warning_state_value}"
+    critical = "${var.free_inodes_critical_state_value}"
+  }
+
+  renotify_interval = "${var.renotify_interval_mins}"
+
+  tags = ["${module.label.tags}"]
+}
+
+resource "datadog_monitor" "disk_freespace" {
+  name    = "Lack of free disk space ${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "Lack of free disk space ${var.freespace_bytes_time} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.instance_id.public_ip}"
+  query   = "avg(last_${var.freespace_bytes_time}):avg:system.disk.free{host:${data.aws_instance.monitored.instance_id}} by {device,host} < ${var.freespace_bytes_critical_state_value}"
```

Comment:
Can you research how we would write this so it works with autoscaling groups? e.g. Terraform won't know the instance id.